citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/AuthorityServiceTest.java

```diff
@@ -196,4 +198,37 @@ public void test_findByUri_noResults() {
         List<IAuthorityEntry> actual = managerToTest.findByUri(user, uri);
         Assert.assertTrue(actual.isEmpty());
     }
+    
+    @Test
+    public void test_create() {
+        String username = "user";
+        OffsetDateTime createdOn = OffsetDateTime.now();
```

Comment:
That's a confusing name, this is not the time the entry was created on, so why is it called that?